src/middlewared/middlewared/api/v25_04_0/virt_instance.py

```diff
@@ -114,6 +116,7 @@ class VirtInstanceUpdate(BaseModel, metaclass=ForUpdateMetaclass):
     cpu: str | None = None
     memory: MemoryType | None = None
     vnc_port: int | None = Field(ge=5900, le=65535)
+    enable_vnc: bool
```

Comment:
Why would we not treat this the same as `vnc_port` attribute and always populate it? It can simplify our checks in the validate method.